soyjs/generator.go

```diff
@@ -8,9 +8,37 @@ import (
 	"github.com/robfig/soy/template"
 )
 
+// The JSFormatter interface allows for callers to choose which
+// version of Javascript they would like soyjs to output. To
+// maintain backwards compatibility, if no JSFormatter is specified
+// in the Options, soyjs will default to the ES5Formatter implemented
+// in exec.go
+//
+// Template returns two values, the name of the template to save
+// in the defined functions map, and how the function should be defined.
+//
+// Call returns two values, the name of the template to save
+// in the called functions map, and a string that is written
+// into the imports
+//
+// Directive takes in a PrintDirective and returns a string
```

Comment:
These godoc go on the method being documented, rather than all at the type level